packages/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -27,16 +27,16 @@ export interface DropdownProps extends MenuProps, OUIAProps {
   children?: React.ReactNode;
   /** Classes applied to root element of dropdown. */
   className?: string;
-  /** Renderer for a custom dropdown toggle. Forwards a ref to the toggle. */
-  toggle: (toggleRef: React.RefObject<any>) => React.ReactNode;
+  /** Dropdown toggle. The toggle should either be a renderer function which forwards the given toggle ref, or a direct ReactNode that should be passed along with the toggleRef property. */
+  toggle: React.ReactNode | ((toggleRef: React.RefObject<any>) => React.ReactNode);
+  /** Ref of the dropdown toggle. This property should be passed when toggle is assigned a node, to enable built in basic interaction handling. */
+  toggleRef?: React.RefObject<HTMLButtonElement>;
```

Comment:
So when I said grouped together  in one props.  So prop that would be an objet of toggle with type react node and toggle ref.  Sou you would either use the toggle prop or the prop with the object.

Reply:
I don't think we've really used that pattern much, but I can update to it if we'd rather having a single prop.

Reply:
Pushed the update

Reply:
We have a few other places where we group related props e.g.
https://github.com/patternfly/patternfly-react/blob/2da278eeebfb3475c31e3693435a8c7f04a0edde/packages/react-core/src/components/Button/Button.tsx#L31-L38
then we use it  here
https://github.com/patternfly/patternfly-react/blob/2da278eeebfb3475c31e3693435a8c7f04a0edde/packages/react-core/src/components/Button/Button.tsx#L85-L86

nit.  Can you define the interface for the group props for consistency.

Reply:
Ah that's actually probably why it looked a bit off to me, since we define the interface outside the props rather than inline. Will update!